src/hb-subset-input.cc

```diff
@@ -117,3 +118,19 @@ hb_subset_input_drop_hints (hb_subset_input_t *subset_input)
 {
   return &subset_input->drop_hints;
 }
+
+/**
+ * hb_subset_input_drop_ot_layout:
+ * @subset_input: a subset_input.
+ *
+ * If enabled ot layout tables will be dropped as part of
+ * the subsetting operation. Currently this defaults to
+ * true.
+ *
+ * REPLACEME
```

Comment:
`Since: REPLACE` would be better for the next time :)

Reply:
Fixed.